addons/im_livechat/static/src/new/core/livechat_service.js

```diff
@@ -59,35 +60,29 @@ export class LivechatService {
         this.rule = init.rule;
     }
 
-    async _createSession() {
-        // TODO - MISSING PREVIOUS OPERATOR ID IN RPC
+    _updateSessionCookie(session) {
+        this.cookie.setCookie(this.SESSION_COOKIE, JSON.stringify(session), 60 * 60 * 24); // 1 day cookie.
+        if (session?.operator_pid) {
+            this.cookie.setCookie(this.OPERATOR_COOKIE, session.operator_pid[0], 7 * 24 * 60 * 60); // 1 week cookie.
+        }
+    }
```

Comment:
This function is not worth introducing: used only once, and the code is small and pretty straight-forward (`setCookie` is an update, named extraction is unnecessary)